### PR TITLE
MueLu: add quality measure as data field

### PR DESCRIPTION
**Pull request by maxfirmbach**
@trilinos/muelu

## Motivation

Adds the aggregate quality measure as additional data field to the aggregate export and enables the visualization in e.g. ParaView.

## Testing

Added some visualization tests including the quality measure export.

### CODE REVIEW

File: packages/muelu/doc/UsersGuide/masterList.xml [hunk 1]
```diff
@@ -876,6 +876,14 @@
       <comment-ML>parameter not existing in ML</comment-ML>
     </parameter>
 
+    <parameter>
+      <name>aggregation: output file: aggregate qualities</name>
+      <type>bool</type>
+      <default>false</default>
+      <description>Wheater to plot the aggregate quality.</description>
```
Comment (cgcgcg) [inline packages/muelu/doc/UsersGuide/masterList.xml:883]: spelling

File: packages/muelu/doc/UsersGuide/paramlist_hidden.tex [hunk 2]
```diff
@@ -183,6 +183,8 @@
         
 \cbb{aggregation: output file: build colormap}{bool}{false}{Whether to output a random colormap in a separate XML file.}
         
+\cbb{aggregation: output file: aggregate qualities}{bool}{false}{Wheater to plot the aggregate quality.}
```
Comment (cgcgcg) [inline packages/muelu/doc/UsersGuide/paramlist_hidden.tex:186]: ditto

Comment (maxfirmbach) [reply packages/muelu/doc/UsersGuide/paramlist_hidden.tex]: Fixed.

File: packages/muelu/doc/UsersGuide/masterList.xml (hunk 1 above)
Comment (maxfirmbach) [reply packages/muelu/doc/UsersGuide/masterList.xml]: Fixed.

**Review by cgcgcg — APPROVED** [review]

**Pull request merged**